Alerting docs: update Alerting Provisioning (#83376)
grafana/grafana: docs/sources/alerting/set-up/provision-alerting-resources/_index.md

   {{< admonition type="note" >}}
   The JSON output from the majority of Alerting HTTP endpoints isn't compatible for provisioning via configuration files.

   If you need the alerting resources for file provisioning, then use the [Export Alerting endpoints](/docs/grafana/<GRAFANA_VERSION>/alerting/set-up/provision-alerting-resources/export-alerting-resources#export-api-endpoints) to return or download them in provisioning format.
   {{< /admonition >}}

## Export alerting resources

> You can export both manually created and provisioned alerting resources. Additionally, it's possible to edit and export an alert rule without applying the changes.

Reviewer: ```suggestion
You can export both manually created and provisioned alerting resources. You can also edit and export an alert rule without applying the changes.
```
